fs/fshttp/http.go

```diff
@@ -72,16 +72,20 @@ func NewTransportCustom(ctx context.Context, customize func(*http.Transport)) ht
 		t.TLSClientConfig.Certificates = []tls.Certificate{cert}
 	}
 
-	// Load CA cert
-	if ci.CaCert != "" {
-		caCert, err := os.ReadFile(ci.CaCert)
-		if err != nil {
-			log.Fatalf("Failed to read --ca-cert: %v", err)
-		}
+	// Load CA certs
+	if ci.CaCert != nil && len(ci.CaCert) != 0 {
+
 		caCertPool := x509.NewCertPool()
-		ok := caCertPool.AppendCertsFromPEM(caCert)
-		if !ok {
-			log.Fatalf("Failed to add certificates from --ca-cert")
+
+		for i := 0; i < len(ci.CaCert); i++ {
+			caCert, err := os.ReadFile(ci.CaCert[i])
+			if err != nil {
+				log.Fatalf("Failed to read --ca-cert file %s : %v", ci.CaCert[i], err)
+			}
+			ok := caCertPool.AppendCertsFromPEM(caCert)
+			if !ok {
+				log.Fatalf("Failed to add certificates from --ca-cert")
```

Comment:
This should probably have the file name in too so the user can track down which certs are the problem.

    log.Fatalf("Failed to add certificates from --ca-cert %q", cert)